src/test/java/common/ValueChoosers.java

```diff
@@ -0,0 +1,35 @@
+package common;
+
+import com.github.javafaker.Faker;
+import org.apache.commons.lang3.RandomUtils;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+
+public class ValueChoosers {
+
+    public static String setRandomSex() {
+        ArrayList<String> sexes = new ArrayList<>();
+        sexes.add("M");
+        sexes.add("F");
+        return sexes.get(RandomUtils.nextInt(0, sexes.size()));
+    }
+
+    public static String newSex(String oldSex) {
+        String newSex = "M";
+        if (oldSex.equals("M")) {
+            newSex = "F";
+        }
+        return newSex;
+    }
+
+    public static String setDob() {
+        return RandomUtils.nextInt(1900,2020) + "-" + RandomUtils.nextInt(10, 12) + "-" + RandomUtils.nextInt(10, 31);
```

Comment:
Za ovo imas Dates klasu gde bukvalno mozes da mu kazes da ide minusDays i to posle mozes da formatiras u kom formatu zelis sa DateTimeFormaterom, istrazi malo o tome :)